src/utils.js

```diff
@@ -17,11 +16,17 @@ export function getServiceName(obj) {
 		throw new Error(`getServiceName must be called with a string url or with a SystemJS load object that has an address`);
 	}
 
+	const windowPath = hasWindow
+		? window.location.pathname.substring(0, window.location.pathname.lastIndexOf('/') + 1)
+		: '';
+
 	// The service name might have a path in it!
 	// For example, you might `import a from 'service/a/path.js!sofe';`
 	// In that case, we want `getServiceName` to return just `'sesrvice'`;
 	let urlParts = getRegex().exec(address);
-	const splits = urlParts[5].split('/');
+	const servicePath = urlParts[5];
+
+	const splits = servicePath.substring(windowPath.length).split('/');
```

Comment:
Why is `window.location.pathname` involved when determining a service name?

Reply:
I believe that this had to do with the loads that SystemJS would pass to the plugin could include location path stuff, but for our use-case it really doesn't matter.